packages/core/src/page/renderer/ClientPageRenderer.js

```diff
@@ -78,6 +78,7 @@ export default class ClientPageRenderer extends AbstractPageRenderer {
 
         controller.setMetaParams(pageState);
         this._updateMetaAttributes(controller.getMetaManager());
+        controller.getMetaManager().clearMetaAttributes();
```

Comment:
We should probably call the `clearMetaAttributes` before calling the `setMetaParams` rather than after, so the values are available in the meta manager for the whole lifecycle of the controller.